glean-core/csharp/GleanTests/Metrics/PingTypeTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System.IO;
+using static Mozilla.Glean.Glean;
+using Xunit;
+
+namespace Mozilla.Glean.Tests.Metrics
+{
+    public class PingTypeTest
+    {
+        public PingTypeTest()
+        {
+            // Get a random test directory just for this single test.
+            string tempDataDir = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+
+            // In xUnit, the constructor will be called before each test. This
+            // feels like a natural place to initialize / reset Glean.
+            GleanInstance.Reset(
+                applicationId: "org.mozilla.csharp.tests",
+                applicationVersion: "1.0-test",
+                uploadEnabled: true,
+                configuration: new Configuration(),
+                dataDir: tempDataDir
+                );
+        }
+
+        [Fact]
+        public void SendCustomPings()
+        {
+            Private.PingType<Private.NoReasonCodes> customPing =
```

Comment:
Please make this test empty, as the other tests, since this is actually not testing the ping as the original one.